src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
```

Comment:
This method takes an array of objects and creates a list from that array.  Then to create multiple records in a database table, you can use a for loop to loop through a list of objects and execute a create statement for each object.Then the same way to delete...
 List<Manufacturer> manufacturers = Arrays.asList(
                new Manufacturer(null, "Lincoln", "USA"),
                new Manufacturer(null, "Ford", "USA"),
                new Manufacturer(null, "Audi", "Germany"),
                new Manufacturer(null, "BMW", "Germany")
        );

        for (Manufacturer manufacturer : manufacturers) {
            manufacturerDao.create(manufacturer);
        }

        List<Long> idsToDelete = Arrays.asList(16L, 22L, 24L, 27L);
        for (Long aLong : idsToDelete) {
            manufacturerDao.delete(aLong);
        }